packages/extension-chrome/src/services/backend/addressStorage.ts

```diff
@@ -0,0 +1,140 @@
+import { bytes } from '@ckb-lumos/codec';
+import { KeystoreService } from '@nexus-wallet/types';
+import { Script } from '@ckb-lumos/base';
+import { Backend } from './backend';
+import { toScript } from './utils';
+
+const MAX_ADDRESS_GAP = 20;
+
+export type AddressInfo = {
+  path: string;
+  addressIndex: number;
+  depth?: number;
+  pubkey: string;
+  blake160: string;
+  lock: Script;
+};
```

Comment:
```suggestion
export type LockInfo = {
  // hd wallet path
  path: string;
  // the BIP44's {@link https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#index index} 
  addressIndex: number;
  publicKey: string;
  blake160: string;
  lock: Script;
};
```

- Address is with network info, but this struct does not seem to need network info
- the `publicKey` is already in used in the other services